src/org/labkey/test/tests/ExpTest.java

```diff
@@ -148,7 +149,7 @@ public void testSteps()
         _ext4Helper.clickExt4Tab("Data");
         waitForText(WAIT_FOR_JAVASCRIPT, "editedCreated");
         dateFormat = new SimpleDateFormat("ddd MMM dd yyyy");
```

Comment:
Ah, re-using dateFormat here was briefly confusing; I looked up the page and saw its initial declaration and wondered how it could match that date string with a format expression of yyyy-MM-dd
My thought was that the reason for the failure might have been an unexpected change in server date format